### PR TITLE
Make the window taller to make it easier to scroll

### PR DESCRIPTION
**Pull request by manan19**

### CODE REVIEW

File: content/productivity/repeat-attack/plugin.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ function AttackList() {
   const [actions, setActions] = useState(op.actions);
 
   let actionList = {
-    maxHeight: "70px",
+    maxHeight: "700px",
```
Comment (phated) [inline content/productivity/repeat-attack/plugin.js:114]: How big of a monitor does this assume? If plugin windows overflow the game boundaries, things get really weird.

Comment (manan19) [reply content/productivity/repeat-attack/plugin.js:114]: which monster plays on a window less than 700 px 😱

Also, this is shorter than my plugin list right now lol

Comment (thomasvbeek) [reply content/productivity/repeat-attack/plugin.js:114]: How about using something like vh? To make it use a percentage of browser window height, combining that with a min-height in pixels can result in something more scalable.

Comment (jacobrosenthal) [reply content/productivity/repeat-attack/plugin.js:114]: @manan19 can we do maybe 300?, i think thats a little better than what it was, its also possible to wire a button to increase decrease :)

Comment (phated) [reply content/productivity/repeat-attack/plugin.js:114]: I kinda liked the vh idea, could do like 50vh which is 50% of your window. @jacobrosenthal thoughts?

Comment (jacobrosenthal) [reply content/productivity/repeat-attack/plugin.js:114]: fine with me @manan19 ?

Comment (jacobrosenthal) [reply content/productivity/repeat-attack/plugin.js:114]: ping @manan19 hope you're recovered. You read to push this over the line?

**Pull request merged**